test/linter/test-spec-urls.ts

```diff
@@ -50,6 +50,8 @@ const specsExceptions = [
 
   // Remove if https://github.com/w3c/browser-specs/pull/667#issuecomment-1200089758 is resolved
   'https://w3c.github.io/device-memory',
+
+  'https://wicg.github.io/directory-upload/proposal.html',
```

Comment:
This would need an issue filed. I'm not sure this spec is going anywhere though, how does it relate to https://fs.spec.whatwg.org/#api-filesystemdirectoryhandle?